vxportia/router.py

```diff
@@ -0,0 +1,92 @@
+
+from vumi.errors import DispatcherError
+from vumi.dispatchers.base import BaseDispatchRouter
```

Comment:
Yes. :/

On the bright side, the new stuff is quite a bit nicer. One just needs to implement three methods:
```
def process_inbound(self, config, msg, connector_name):
        ...
        return self.publish_inbound(msg, new_connector_name, 'default')

    def process_outbound(self, config, msg, connector_name):
        ...        
        return self.publish_outbound(msg, new_connector_name, 'default')

    def process_event(self, config, event, connector_name):
        ...
        return self.publish_event(event, new_connector_name, 'default')
```

The base routing table dispatcher at the bottom of `vumi.dispatchers.endpoint_dispatchers` is a good example.

P.S. I have no idea why the module is called `endpoint_dispatchers`. It's more like `connector_dispatchers`. :/